src/utils/utils.js

```diff
@@ -1,88 +1,112 @@
-exports.isNode = function() {
-	return typeof process !== 'undefined' && process.toString() === '[object process]';
-};
+/**
+ * A regular expression that matches whitespace on either side, but
+ * not in the center of a string
+ *
+ * @type {RegExp}
+ */
+var TRIM_REGULAR_EXPRESSION = /^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g;
+
+/**
+ * Used in typeof comparisons
+ *
+ * @type {String}
+ */
+var OBJECT = 'object';
 
+/**
+ * True if environment is node, false if it's a browser
+ * This seems somewhat inelegant, if anyone knows a better solution,
+ * let's change this (must identify browserify's pseudo node implementation though)
+ *
+ * @public
+ * @type {Boolean}
+ */
+exports.isNode = typeof process !== 'undefined' && process.toString() === '[object process]';
```

Comment:
Explanation: Didn't need to be a function that's evaluated everytime nextTick is called